std_msgs/String frame_a
std_msgs/String frame_b
---
geometry_msgs/Pose pose_a_b
